### ISSUE TITLE
NVDA doesn't read information in edit box in Chrome with mouse hover unless focussed

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#14953]
### Steps to reproduce:

First, ensure NVDA's mouse tracking is enabled with NVDA+m if needed.

Example A:
1) Open the NV Access donation page: https://www.nvaccess.org/support-us/#donation-support
2) Navigate down to the "A Unique $50 / month" option.
3) Hover over the different parts of text.

Example B:
1) Open NVDA's issue tracker here on GitHub: https://github.com/nvaccess/nvda/issues
2) Hover the mouse over the issue search edit (with "is:issue is:open" text)

### Actual behavior:

In both examples above, NVDA reads static text, such as "a unique", the dollar sign or the donation button on the NV Access site, or the number of open and closed issues and filter button on GitHub.

In both examples, the text in the edit box (the 50 by default on the NV Access site, and "is:issue is:open" on GitHub), is not read when hovering over the text.

### Expected behavior:

NVDA should read all information when hovering over it.

#### NVDA version:

2023.1 installed

#### Windows version:

Windows 11 (64-bit) Version: 22H2, Build: 22621.1702 

#### Name and version of other software in use when reproducing the issue:

Chrome Version: 113.0.5672.127 

#### Other information about your system:

This works with Firefox version: 113.0.2 on both sites.

#### Does the issue still occur after restarting your computer?

Yes

#### If NVDA add-ons are disabled, is your problem still occurring?

Yes
